examples/chip-tool/commands/common/ModelCommand.cpp

```diff
@@ -46,11 +46,22 @@ bool isGlobalCommand(uint8_t frameControl)
 
 void ModelCommand::OnConnect(ChipDeviceController * dc)
 {
+    mExchangeContext = dc->GetExchangeManager()->NewContext(mRemoteId, this);
```

Comment:
Client (AKA controller) needs to manually create an ExchangeContext to send/receive messages. (ModelCommand)

Reply:
Yes, before initiating a data model transaction (which maps to a message layer exchange), you have to create an ExchangeContext.  This part we agree on.